folio-service-tools-spring-dev/src/main/java/org/folio/spring/tools/kafka/FolioKafkaProperties.java

```diff
@@ -1,20 +1,25 @@
 package org.folio.spring.tools.kafka;
 
-import java.util.List;
-import java.util.Map;
 import lombok.AllArgsConstructor;
 import lombok.Data;
 import lombok.NoArgsConstructor;
 import org.springframework.boot.context.properties.ConfigurationProperties;
 import org.springframework.stereotype.Component;
 
+import java.util.List;
+import java.util.Map;
+
 /**
  * Application properties for kafka message consumer.
  */
 @Data
 @Component
 @ConfigurationProperties("folio.kafka")
 public class FolioKafkaProperties {
+  /**
+   * Kafka header name for tenant_id
+   */
+  public static final String TENANT_ID = "folio.tenantId";
```

Comment:
Why can't we use the same name as for the okapi header `x-okapi-tenant` ?

Reply:
folio.tenantId is used in folio-kafka-wrapper project as well. Also x-okapi-tenant is HTTP web specific.